resources/wiki-overrides.properties

```diff
@@ -47,9 +41,6 @@ prereq-buildstep=http://wiki.jenkins-ci.org/display/JENKINS/Prerequisite+build+s
 project-health-report=https://wiki.jenkins-ci.org/display/JENKINS/Project+Health+Report+Plugin
 puppet=https://wiki.jenkins-ci.org/display/JENKINS/Puppet+Plugin
 ruby-runtime=https://wiki.jenkins-ci.org/display/JENKINS/Ruby+Runtime+Plugin
-serenity=https://wiki.jenkins-ci.org/display/JENKINS/Serenity+Plugin
-## Currently has the <url> tag, BUT with a typo http://http://wiki.jenkins-ci.org/display/JENKINS/Skytap+Cloud+CI+Plugin
-skytap=https://wiki.jenkins-ci.org/display/JENKINS/Skytap+Cloud+CI+Plugin
```

Comment:
This is an interesting case because it used to be wrong to do this: https://github.com/jenkinsci/skytap-cloud-plugin/commit/c7bf2fe43991a0527d003e5e1e8ad7f02d5839ca

Now it's not, but the wiki page is still needed because the repo only contains placeholder docs — but redirects to the plugin site 👎  So this will make all of the documentation inaccessible.

Reply:
There are two <url> tags within the POM and the one outside of <scm> takes precendence -- I think this should make no difference for Skytap

Reply:
Oh right, I failed at reading XML. Sorry about that.

The URL was set in https://github.com/jenkinsci/skytap-cloud-plugin/commit/ccfe470652a023433c59d0b99ce36bb658a8cdb8 but is linking to plain HTTP. So the original problem doesn't exist anymore, but a different one.

Reply:
https://github.com/jenkins-infra/update-center2/blob/f0fff816539341fdfbf2ad206556f5f1dec12858/src/main/java/io/jenkins/update_center/HPI.java#L406 should take care of that though, so I'm fine with this change.